server/tserver/src/main/java/org/apache/accumulo/tserver/memory/LargestFirstMemoryManager.java

```diff
@@ -147,6 +148,11 @@ protected boolean tableExists(TableId tableId) {
     return context.getTableConfiguration(tableId) != null;
   }
 
+  private boolean tableBeingDeleted(TableId tableId) {
+    var state = context.getTableManager().getTableState(tableId);
+    return state.equals(TableState.DELETING);
```

Comment:
Should use `==` to compare enum.

```suggestion
    return context.getTableManager().getTableState(tableId) == TableState.DELETING;
```